e2e_playwright/hostframe_app_test.py

```diff
@@ -75,6 +75,66 @@ def test_handles_host_theme_message(
     )
 
 
+def test_handles_set_file_upload_client_config_message(iframed_app: IframedPage):
```

Comment:
great test!
one small nit: would it be possible to somehow verify that the file_content was indeed uploaded successfully? Maybe by reading the content you are setting here in the test app and write the content to the screen and grab for it? Then we would have a complete test!

Reply:
Yes, I think it would be a good idea to add tests for file_uploader functioning in the host frame, although I think it should be a separate test; for this one, we check mostly that the file upload client config is applied to URLs and headers. 

For now I just added check for file_uplaoder upload request without config that it uploads correctly (with status code). 

I am happy to follow up with that tests as the separate PR!